.eslintrc

```diff
@@ -0,0 +1,48 @@
+{
+    "parser": "babel-eslint",
+    "settings": {
+        "react": {
+            "version": "detect" // React version. "detect" automatically picks the version you have installed.
+        }
+    },
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
```

Comment:
I would suggest to remove any reference to React plugins here, and replace them with `"plugin:vue/recommended"`. You can see eslint failing by running `npm run serve`.

Reply:
@francescozaia All the suggested changes have been taken care of in the latest commit, thanks for pointing out!